lua/fpp/server/ownability.lua

```diff
@@ -499,11 +499,15 @@ function FPP.RecalculateConstrainedEntities(players, entities)
 end
 
 local entMem = {}
-local function constraintRemovedTimer(ent1, ent2, constrainedEnts)
-    if not IsValid(ent1) and not IsValid(ent2) or not constrainedEnts then return end
-
-    FPP.RecalculateConstrainedEntities(player.GetAll(), constrainedEnts)
+local function constraintRemovedTimer(ent1, ent2)
+    if not IsValid(ent1) and not IsValid(ent2) then return end
     entMem = {}
+
+    local constrainedEnts1 = constraint.GetAllConstrainedEntities(ent1)
+    FPP.RecalculateConstrainedEntities(player.GetAll(), constrainedEnts1)
+
+    local constrainedEnts2 = constraint.GetAllConstrainedEntities(ent2)
+    FPP.RecalculateConstrainedEntities(player.GetAll(), constrainedEnts2)
```

Comment:
I think it's quite possible to have overlap between `constrainedEnts1` and `constrainedEnts2`, especially if somewhere there is a different constraint that keeps the whole contraption together.

You could build the union of the two tables, but I have a different idea (see next post)